### ISSUE TITLE
Ability to select allies houses to get info about thier resources in storage/barracks etc

### ISSUE THREAD

**Issue by reyandme** [Kromster80/kam_remake#204]
Sounds very natural and usefull.

Are there any barriers, why should not we do it ?

**Comment by lewinjh** [comment]
Potentially confusing if you think you have your own barracks/storehouse selected. Not opposed to the idea, but we'd need to have a clear distinction between selecting your own stuff and your allies' (e.g. interface greyed out)

**Comment by reyandme** [comment]
> we'd need to have a clear distinction between selecting your own stuff and your allies' (e.g. interface greyed out)

Yep. Also we can distinguish houses of different allies (f.e. with some color mark or outline on the house interface).

**Comment by thunder6** [comment]
Hello guys!
An easy solution could be if simply would disable the units selection area with the train and paging buttons, so basicly the remain thing would be only the Weapon Table. This is enough sign for a player it is not one of my barracks. Maybe using a picture which  disables/ make those buttons unvisible there?
Other houses don't need this function because the wares are visible in their store. 
Bests!
t

**Comment by Kromster80** [comment]
Idea sounds good.
Interactive elements could be grayed or semi-transparent.

**Comment by reyandme** [comment]
Implemented in my repo
